backend/src/main/java/com/benine/backend/http/CreatePresetHandler.java

```diff
@@ -41,19 +46,28 @@ public void handle(String s, Request request, HttpServletRequest req, HttpServle
 
       CameraController cameraController = ServerController.getInstance().getCameraController();
       Camera camera = cameraController.getCameraById(Integer.parseInt(camID));
-
+      String tags= request.getParameter("tags");
+      List<String> tagList = null;
+      if (tags != null) {
```

Comment:
You can clean this up by initializing tagList as Arrays.asList("none") and changing it if tags != null.